src/evm.rs

```diff
@@ -813,7 +814,7 @@ impl<'a, Ext, Db: Database + DatabaseCommit> EvmNeedsBlock<'a, Ext, Db> {
 
 impl<'a, Ext, Db: Database + DatabaseCommit, TrevmState: HasBlock> Trevm<'a, Ext, Db, TrevmState> {
     /// Run a function with the provided block, then restore the previous block.
-    pub fn with_block<F, B, NewState>(mut self, f: F, b: &B) -> Trevm<'a, Ext, Db, NewState>
```

Comment:
this is a breaking change, so we'll go to 0.5.0. can you update `with_cfg` and any other functions that take a closure to match?